.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: objective-c
 osx_image: xcode8.3
 rvm:
-  - 2.3.0
+  - 2.4.0
```

Comment:
Should be updated to 2.4.1, there are many bugs in 2.4.0 - which may cause your builds to randomly fail.